### PR TITLE
[feat]: [PIE-3635]: Add invalid yaml to InvalidYamlException

### PR DESCRIPTION
**Pull request by namanvermah**
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/32706)

### CODE REVIEW

**Comment by namanvermah** [comment]
trigger messagecheck

**Review by mohitg0795 — APPROVED** [review]
> Reviewed 3 of 3 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/32706)* status: 3 file(s) to review. 1 of 2 approvals obtained.

File: 800-pipeline-service/src/main/java/io/harness/pms/pipeline/service/PMSPipelineServiceHelper.java [hunk 1]
```diff
@@ -166,7 +167,7 @@ public GovernanceMetadata validatePipelineYamlAndSetTemplateRefIfAny(
                   YamlSchemaErrorDTO.builder().message(ex.getMessage()).fqn("$.pipeline").build()))
               .build();
       throw new io.harness.yaml.validator.InvalidYamlException(
-          HarnessStringUtils.emptyIfNull(ex.getMessage()), ex, errorWrapperDTO);
+          HarnessStringUtils.emptyIfNull(ex.getMessage()), ex, errorWrapperDTO, pipelineEntity.getData());
```
Comment (abhinav-harness) [inline 800-pipeline-service/src/main/java/io/harness/pms/pipeline/service/PMSPipelineServiceHelper.java:170]: ExceptionUtils.getMsg

**Review by brijeshdhakar — APPROVED** [review]
> Reviewed 3 of 3 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/32706)* status: :shipit: complete! ✓

**Pull request merged**